internal/builder/templates.go

```diff
@@ -140,14 +140,14 @@ patches:
       path: /spec/template/spec/containers/0/args/-
       value: --watch-label-selector=!{{.ShardingKey}}
 {{- end }}
-{{- if and .SupportsObjectLevelWorkloadIdentity .EnableObjectLevelWorkloadIdentity }}
+{{- if .SupportsObjectLevelWorkloadIdentity }}
```

Comment:
What's the reason for generating this patch if the gate is set to its default value `false`?

Reply:
> I added a commit to fix the feature gate not being rendered when disabled. This is important for keeping the `FluxInstance` API consistent in case this feature gate becomes opt-out in CNCF Flux the future.